docs/participate/support-the-network/run-a-node.md

```diff
@@ -4,18 +4,48 @@ description: Everything you need to know and learn to setup an IOTA Mainnet or D
 ---
 
 
-# Heading 1 (Page Title)
+# Run a Node
 
-content - introduction to the topic
+One way to support the IOTA project and community is to run your own IOTA node. This is also an excellent way to get into the basic concepts of IOTA (and to a certain degree of Distributed Ledger Technologies in general).
 
 
-## Heading 2 (Sub topic 1)
+## What is a node?
 
-content 
+Nodes are the backbone of distributed networks. In the IOTA case nodes specifically accept requests from participants (e.g. transactions from wallets), verify transactions and communicate among each other in case of unclear situations. This ultimately leads to a consensus and thus to a consistent ledger state for all nodes in the network. 
 
-## Heading 2 (Sub topic 2)
+By having more active nodes in the network it grows more robust and less centralised. On the personal side you may feel more secure when sending transaction-requests to your own node. Keep in mind that - in contrast to other DLT - there is no direct or indirect financial incentive to run your own node. (Hint at Mana?)
```

Comment:
I think we shouldn't mention Mana in regard to financial incentives. We don't know if there ever will be a marked and imo the wiki should be mostly fact based :)

Reply:
Good point, I'm perfectly fine with removing that :)